docs/docs/getting-started-devs.md

```diff
@@ -2,34 +2,42 @@
 
 *If you don't want to code anything and only use Jelly with your Apache Jena/RDF4J application, see [the dedicated guide](getting-started-plugins.md) about using Jelly-JVM as a plugin.*
 
-This guide explains a few of the basic functionalities of Jelly-JVM and how to use them in your code. Jelly-JVM is written in Scala, but it [can be used from Java as well](#quick-start-java-titanium-rdf-api).
+This guide explains a few of the basic functionalities of Jelly-JVM and how to use them in your code. Jelly-JVM is mainly written in Java, but some modules [are written entirely in Scala](user/reactive.md), as well as unit and integration tests.
 
-## Quick start – Scala & Apache Jena
+## Quick start – Scala or Java & Apache Jena
 
 Depending on your RDF library of choice (Apache Jena, RDF4J, Titanium), you should import one of the dependencies: `jelly-jena`, `jelly-rdf4j`, [`jelly-titanium-rdf-api`](user/titanium.md)[^1]. In our examples we will use Jena, so let's add this to your `build.sbt` file:
 
 ```scala title="build.sbt"
 lazy val jellyVersion = "{{ jvm_package_version() }}"
 
 libraryDependencies ++= Seq(
-  "eu.ostrzyciel.jelly" %% "jelly-jena" % jellyVersion,
+  "eu.neverblink.jelly" %% "jelly-jena" % jellyVersion,
 )
 ```
 
 If you are working with Maven, add this to your `pom.xml`:
 
 ```xml
 <dependency>
-    <groupId>eu.ostrzyciel.jelly</groupId>
-    <artifactId>jelly-jena_3</artifactId>
+    <groupId>eu.neverblink.jelly</groupId>
+    <artifactId>jelly-jena</artifactId>
     <version>{{ jvm_package_version() }}</version>
 </dependency>
 ```
 
+If you are using Gradle, add this to your `build.gradle`:
+
+```groovy
+dependencies {
+    implementation "eu.neverblink.jelly:jelly-jena:${jellyVersion}"
+}
+```
```

Comment:
If we are doing all this (maven, sbt, gradle), maybe use a tabbed structure in the docs, to make this a bit more compressed? See: https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks

Please put Maven first.